components/GalleryPictures.tsx

```diff
@@ -20,15 +20,29 @@ function GalleryPictures({}: Props) {
         alt=""
       />
 
-      <div>
-        <h4>Kitchens</h4>
-        <p>From modern to traditional</p>
-        <div>
-          {/* Cabinet Style */}
-          {/* Cabinet Style */}
-          {/* Cabinet Style */}
+      <div className="px-0 md:px-10">
+        <h4 className="text-4xl font-light">Kitchens</h4>
+        <p className="font-bold text-2xl mt-1">From modern to traditional</p>
+        <div className="flex space-x-2 my-2">
+          <img
+            className="h-10 w-10 rounded-full"
+            src="https://i.gyazo.com/117b1b0999fd54f99495be9a7e9d7419.png"
+            alt=""
+          />
+          <img
+            className="h-10 w-10 rounded-full"
+            src="https://i.gyazo.com/117b1b0999fd54f99495be9a7e9d7419.png"
+            alt=""
+          />
+          <img
+            className="h-10 w-10 rounded-full"
+            src="https://i.gyazo.com/117b1b0999fd54f99495be9a7e9d7419.png"
+            alt=""
```

Comment:
pretty sure you're not supposed to use raw `img` tags in React/NextJS and instead prefer `Image`

Reply:
It's not mandatory to use Image, its just smarter since the img just loads slower. But I plan to fix these towards the end of the project since it's just a quick fix.

Reply:
sure, but why introduce tech debt now rather than do the correct thing from the start, esp. if's easier.  And you're right that interface between the two is pretty similar so can just be a drop-in fix

Reply:
I'm just being a little lazy to get a demo up and running asap.